packages/storage/exp-types/index.d.ts

```diff
@@ -0,0 +1,115 @@
+/**
+ * @license
+ * Copyright 2017 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { FirebaseApp } from '@firebase/app-types-exp';
+import { Observer, Unsubscribe } from '@firebase/util';
+
+export interface FullMetadata extends UploadMetadata {
```

Comment:
The public facing types should be defined in `storage-types` instead. You can create a `exp` folder in `storage-types`

Reply:
Moved.